src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java

```diff
@@ -105,66 +94,23 @@ private void downloadYoloModel() {
     File modelNamesFile = new File(darknetHome + File.separator + modelNames);
 
     if (!modelConfigFile.exists()) {
-      // download & cache!
-      downloadAndCache(modelConfigUrl, modelConfigFile, null);
+      log.error("model does not exists {}", modelConfigFile.getAbsolutePath());
     }
     if (!modelWeightsFile.exists()) {
-      // download & cache!
-      downloadAndCache(modelWeightsUrl, modelWeightsFile, "Large download 200mb +/-");
+      log.error("weights do not exist {}", modelWeightsFile.getAbsolutePath());
     }
     if (!modelNamesFile.exists()) {
-      // download & cache!
-      downloadAndCache(modelNamesUrl, modelNamesFile, null);
+      log.error("names do not exist {}", modelWeightsFile.getAbsolutePath());
     }
-    log.info("downloadYoloModel - end");
-
-  }
-
-  private void downloadAndCache(String uri, File location, String details) {
-    // TODO: clean up the error handling here.
-    log.info("downloadAndCache - begin");
-
-    log.info("Downloading {} to file location {}) {}", uri, location.getAbsolutePath(), details);
-    URL url = null;
-    try {
-      url = new URL(uri);
-    } catch (MalformedURLException e) {
-      log.warn("Invalid url passed! {}", uri);
-      e.printStackTrace();
-      return;
-    }
-    InputStream in = null;
-    try {
-      in = url.openStream();
-    } catch (IOException e) {
-      log.error("Error opening a connection to {} ", uri, e);
-      return;
-    }
-    DataInputStream dis = new DataInputStream(new BufferedInputStream(in));
-    try {
-      // open up the destination file for writing
-      FileOutputStream fos = new FileOutputStream(location);
-      IOUtils.copy(dis, fos);
-      fos.close();
-      dis.close();
-    } catch (IOException e) {
-      log.warn("Error downloading.", e);
-      // clean up a partially written file
-      if (location.exists()) {
-        log.warn("Partially downloaded file.. cleaning up");
-        location.delete();
-      }
-      return;
-    }
-    log.info("downloadAndCache - end");
+    log.info("checkFiles - end");
   }
 
   private void loadYolo() {
     log.info("loadYolo - begin");
 
     // If the model isn't there, we should download it and cache it.
     log.info("Staritng yolo download verification");

```

Comment:
this log message is wrong, it doesn't download anymore..  same with the following log message.  and there's no validation that "checkFiles" was successful.

Reply:
this is dead to me...